configs/cylinder3d/cylinder3d_4xb4_3x_semantickitti.py

```diff
@@ -0,0 +1,37 @@
+_base_ = [
```

Comment:
In the latest version, the batch_size has been changed to 2. We may need to train cylinder3d with batch_size of 2. And also rename the file to 4xb2.

Reply:
But it seems that batch_size of 4 is better than batch_size of 2.